llvm/lib/Transforms/IPO/SampleProfile.cpp

```diff
@@ -455,9 +455,9 @@ class SampleProfileMatcher {
     Unknown = 32,
   };
 
-  // For each function, store every callsite state into a map, of which each
-  // entry is a pair of callsite location and MatchState. This is used for
-  // profile stalness computation and report.
+  // For each function, store every callsite a matching state into this map, of
```

Comment:
store every callsite a matching state into this map, of which each entry is a pair of callsite location and MatchState.
->
store every callsite and its matching state into this map.